src/WebJobs.Script.WebHost/ContainerManagement/LegionContainerInitializationHostedService.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.WebHost.Management;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.ContainerManagement
+{
+    public class LegionContainerInitializationHostedService : LinuxContainerInitializationHostedService
+    {
+        private const string ContextFile = "Context.txt";
+
+        private readonly IEnvironment _environment;
+        private readonly ILogger _logger;
+
+        public LegionContainerInitializationHostedService(IEnvironment environment, IInstanceManager instanceManager,
+            ILogger<LinuxContainerInitializationHostedService> logger, StartupContextProvider startupContextProvider)
```

Comment:
Was expecting the logger would be ILogger<LegionContainerInitializationHostedService> logger now?

Reply:
Fixed!